app/mailers/notifier.rb

```diff
@@ -53,11 +53,12 @@ def statement(args)
     send_nucore_mail args[:user].email, text("views.notifier.statement.subject", facility: @facility)
   end
 
-  def order_detail_status_changed(order_detail_id)
-    @order_detail = OrderDetail.includes(:order, :order_status, :product).find(order_detail_id)
+  def order_detail_status_changed(order_detail)
+    facility = order_detail.facility.abbreviation
```

Comment:
I I think we do need the `@order_detail = order_detail` so it gets passed to the view